### PR TITLE
OneClick default shipping settings (#225)

### PR DESCRIPTION
**Pull request by rus-daniel**

### CODE REVIEW

**Comment by StefBrito** [comment]
The changement by default are not taking apart :
image

image

It should take in consideration the shipments types

File: app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php [hunk 1]
```diff
@@ -122,11 +122,12 @@ protected function getMappingFieldHtml($fieldset, $group)
...
         $path = sprintf('oyst_oneclick/carrier_mapping/%s', $group['value']);
 
-        $data = '0';
-        $inherit = false;
+        $inherit = true;
         if (isset($configData[$path])) {
             $data = $configData[$path];
             $inherit = false;
+        } else {
+            $data = Mage::getStoreConfig($path);
```
Comment (mehdichaouch) [inline app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php]: put this before ```$inherit = true;``` and remove the else

File: app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php [hunk 2]
```diff
@@ -162,11 +163,12 @@ protected function getDelayFieldHtml($fieldset, $group)
...
         $path = sprintf('oyst_oneclick/carrier_delay/%s', $group['value']);
 
-        $data = 48;
-        $inherit = false;
+        $inherit = true;
         if (isset($configData[$path])) {
             $data = $configData[$path];
             $inherit = false;
+        } else {
+            $data = Mage::getStoreConfig($path);
```
Comment (mehdichaouch) [inline app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php]: put this before ```$inherit = true;``` and remove the else

File: app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php [hunk 3]
```diff
@@ -203,11 +205,12 @@ protected function getNameFieldHtml($fieldset, $group)
...
         $path = sprintf('oyst_oneclick/carrier_name/%s', $group['value']);
 
-        $data = $group['label'];
-        $inherit = false;
+        $inherit = true;
         if (isset($configData[$path])) {
             $data = $configData[$path];
             $inherit = false;
+        } else {
+            $data = Mage::getStoreConfig($path);
```
Comment (mehdichaouch) [inline app/code/community/Oyst/OneClick/Block/Adminhtml/System/Config/Form/Fieldset/Carrier/Mapping.php]: put this before ```$inherit = true;``` and remove the else

**Pull request merged**